lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -1328,11 +1329,15 @@ function dosomething_user_info_form($form, &$form_state, $account) {
     '#default_value' => $zipcode,
   );
   $birthday = dosomething_user_get_field_birthdate(dosomething_user_get_field('field_birthdate', $account), 'm/d/Y');
+  $birthDate = explode("/", $birthday);
+  $age = (date("md", date("U", mktime(0, 0, 0, $birthDate[0], $birthDate[1], $birthDate[2]))) > date("md") ? ((date("Y") - $birthDate[2]) - 1) : (date("Y") - $birthDate[2]));
```

Comment:
yeah this looks pretty insane. you should be able to call `diff` on a DateTime object and just grab the value from `y`
like
```
today = new DateTime("now")
bday = new DateTime($birthday_value);
age = bday->diff(today)->year;
```